tests/test.rs

```diff
@@ -432,7 +432,7 @@ fn test_integration() {
         let entry = entry.unwrap();
         let path = entry.path();
 
-        if path.starts_with("tests/basic_zip_zip") || path.starts_with("tests/basic_zip_dir") {
+        if path.starts_with("tests/basic_zip_zip") || path.starts_with("tests/basic_zip_dir") || path.starts_with("tests/java") {
```

Comment:
Nit: you could put the file in test/java/test.java so you wouldn't have to skip the dir here